music21/roman.py

```diff
@@ -832,6 +859,11 @@ def romanNumeralFromChord(chordObj,
         'i64b3': 'Sw43',
         'io6b5b3': 'Ger65',
     }
+    secondaryDominantSwaps = {
+        ('#', 'i'): ('vii', 'ii'),
+        ('#', 'iv'): ('vii', 'V'),
+        ('', 'II'): ('V', 'V'),
```

Comment:
what about the other common secondaries: V/vi? V/iii, and do these work in minor as well as major?

Reply:
Now fully mapped out with major/minor and tests.